requirements.txt

```diff
@@ -1,4 +1,4 @@
-Django==1.11.1
+Django==1.11.16
```

Comment:
Next time we should bump deps in a separate PR as we may want to revert without losing work.

Reply:
I moved it to a new PR #4142